dynamic_rest/viewsets.py

```diff
@@ -336,5 +336,21 @@ def list_related(self, request, pk=None, field_name=None):
         return Response(serializer.data)
 
 
+class BulkCreationMixin(object):
+    """
+    Either create a single or many model instances in bulk by using the
+    Serializer's many=True ability from Django REST >= 2.2.5.
+    """
+    def create(self, request, *args, **kwargs):
+        if not isinstance(request.data, list):
```

Comment:
In DREST, we typically support creation of a single option in two different formats, e.g the basic form that is supported by REST framework:

```
{
  "name": "Fido",
  "age": 2
}
```

but also the form where data is keyed by the serializer name (mostly to support Ember Data, but also because this is a more flexible format that allows for metadata to be passed in request data without conflicting with field names):

```
{
  "dog": {
    "name": "fido",
    "age": 2
  }
}
```

Can you modify this check so that it respects the 2nd form, which in this case should use the plural name of the serializer:

```
{
  "dogs": [{
    "name": "Spot"
  }, {
    "name": "Fido"
  }]
}
```